recipe/meta.yaml

```diff
@@ -56,6 +63,11 @@ build:
   skip: true  # [win]
   # cuda 11.8 was dropped due to maintenance effort, see discussion in #177
   skip: true  # [cuda_compiler_version == "11.8"]
+{% if "rc" in version %}
+  skip: true  # [rc == "None"]
+{% else %}
+  skip: true  # [rc == "rc"]
+{% endif %}
```

Comment:
Correct me if I'm wrong, but I don't see how anything would be built for `2.6.0.rc1`) unless we fully duplicate our variant configs (to have variants for both values of `rc`). Currently, this PR adds
```yaml
rc:
- None
```
to all of them, but that means we'd just skip all builds if someone adds `.rcX` to the version.

Reply:
When one adds an `rc` version, they will need to re-render. Doing so updates the value like so

```yaml
rc:
- rc
```

Reply:
OK, now I finally understand the intended mechanics, thanks.